### PR TITLE
Added CORS and better error handling

### PR DESCRIPTION
**Pull request by jonnyshaw89**
- When the Address Index Api errors the response is now a 500 error
 - CORS has been added so that this can be called from Javascript

### CODE REVIEW

**Comment by insacuri** [comment]
The flask cors documentation says we should have some CSRF protection before using it.  Do we have it enabled somewhere? because I can't see it.  I'm not well versed in this though and we might not need it at all;  Just thought I'd ask the question.

**Comment by jonnyshaw89** [comment]
I cant see the benefit of having CSRF on a stateless API?

File: app/__init__.py [hunk 1]
```diff
@@ -29,6 +31,8 @@ def address_search():
             return "ADDRESS INDEX API CONNECTION TIMED OUT", 503
         except requests.exceptions.ReadTimeout:
             return "ADDRESS INDEX API READ TIMED OUT", 408
+        except Exception as exception:
+            return exception.message, 500
     else:
```
Comment (MebinAbraham) [inline app/__init__.py]: I get `AttributeError: 'Exception' object has no attribute 'message'` if I invalidate the `AUTH_KEY`.

**Comment by jonnyshaw89** [comment]
@MebinAbraham Ready for re-review

**Pull request merged**